lms/templates/seq_module.html

```diff
@@ -7,7 +7,7 @@
     </ul>
 
     <div class="sequence-list-wrapper">
-      <ol role="tablist" aria-label="${_('Section Navigation')}" id="sequence-list">
+      <ol role="tablist" aria-label="${_('Section')}" id="sequence-list">
```

Comment:
The aria-label needs to be on the parent nav element.  Remove aria-label (can't have that on an ```<ol>```) and put it on the parent ```<nav>``` and change the label to "Unit" or "Sequence".  We are using "Section" already on the left/right arrows at the bottom of the page.